jetty-core/jetty-security/src/main/java/org/eclipse/jetty/security/jaas/spi/LdapLoginModule.java

```diff
@@ -731,13 +726,13 @@ public static String convertCredentialLdapToJetty(String encryptedPassword)
 
         if (encryptedPassword.toUpperCase(Locale.ENGLISH).startsWith("{MD5}"))
         {
-            String src = encryptedPassword.substring("{MD5}".length(), encryptedPassword.length());
+            String src = encryptedPassword.substring("{MD5}".length());
             return "MD5:" + base64ToHex(src);
```

Comment:
I'm surprised that our Password or Credential class doesn't have this code in it?

Reply:
Because it's different: apparently, passwords are bound into LDAP attributes as `{MD5}password`, but Jetty's format is `MD5:password`.